src/notification.cpp

```diff
@@ -299,8 +458,10 @@ void Notification::close()
 */
 void Notification::checkActionInvoked(uint id, QString actionKey)
 {
-    if (id == replacesId_ && actionKey == "default") {
-        emit clicked();
+    if (id == replacesId_) {
+        if (actionKey == DEFAULT_ACTION_NAME) {
```

Comment:
Can be folded back like  in the original, but `"default"` replaced with `DEFAULT_ACTION_NAME`?